CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/markers/AbstractMarker.java

```diff
@@ -95,6 +95,7 @@ public List<Integer> getConnectedNodeIdList() {
         return idList;
     }
 
+    // Do nothing here as this gets overidden in child classes
```

Comment:
sorry I probably didn't make this clear, stick this in the function itself (this will probably get flagged by sonar)